README.md

```diff
@@ -3,7 +3,7 @@
 ChangesReloaded is a project developed in context of SWT 2020 based on the Changes Browser project from 2017. It is supposed to simplify the old tool.
 
 ## Installation
-In order to use our tool, you have to install it in your squeak image. You can either do that by using the Git Browser or by using Monticello.
+In order to use our tool, you have to install it in your squeak image. Our preferred way is to use squeak's Git Browser.
```

Comment:
```suggestion
In order to use our tool, you have to install it in your squeak image. Our preferred way is to use Squeak's GitBrowser.
```

Reply:
well. no. if you wanted to change this you had to change it everywhere. everywhere else it is "squeak image" and "Git Browser"